src/roswire/proxy/file.py

```diff
@@ -186,6 +186,28 @@ def islink(self, path: str) -> bool:
         code, output, duration = self.__shell.execute(cmd)
         return code == 0
 
+    def find(self,
+             path: str,
+             filename: str
+             ) -> List[str]:
+        """
+        Returns a list of files that match the provided filename
+        in a given directory recursively.
+
+        Parameters:
```

Comment:
This is good, but not quite the right format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Is this format different than what you've used in other functions of this class?

Reply:
Apparently I've messed up the documentation in this module. Don't worry about it. I'll create an issue to look at the documentation later.